controller/controller.go

```diff
@@ -51,7 +51,7 @@ import (
 	"k8s.io/client-go/tools/record"
 	ref "k8s.io/client-go/tools/reference"
 	"k8s.io/client-go/util/workqueue"
-	glog "k8s.io/klog"
+	glog "k8s.io/klog/v2"
```

Comment:
eventually we should clean this up...

Reply:
https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/issues/96